### DISCUSSION TITLE
Url prefix

### DISCUSSION THREAD

**Discussion by jfayot** [diced/zipline#548] [Ideas]
Is it possible to use zipline behind reverse nginx proxy with a url prefix?

**Comment by PBOwner** [comment]
url prefix? a subdomain you mean?

**Reply by jfayot** [reply]
No, sorry for the wording. I meant base url. I'd like something like this to work:

```
location /my-custom-prefix/ {
  proxy_pass http://localhost:3000;
  proxy_set_header Host $host;
  proxy_set_header X-Real-IP $remote_addr;
  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_set_header X-Forwarded-Proto $scheme;
}
```

Currently, this is redirecting me to /

**Reply by diced** [reply]
you can't use a url prefix, zipline only supports it being on /. I don't have any plans to change this either, sorry 😐
